DSpace/dspace-angular
# Fix TypeError on search pages

## References
fixes [#897](https://github.com/DSpace/dspace-angular/issues/897)

## Description
This PR resolves the issue of the " ERROR TypeError: Cannot read property 'uuid' of undefined" in the console found on the search page when auto completing the search facets.

## Instructions for Reviewers
To verify the fix, perform the following steps: 
* Go to the search page, enter values in a search facet
* Verify that the autocomplete works.
* Verify that no errors are thrown in the console. 
* Navigate to the item edit page
* Edit metadata 
* Verify that the metadata field autocomplete works

## Checklist
